include/extractor/compressed_edge_container.hpp

```diff
@@ -58,21 +59,19 @@ class CompressedEdgeContainer
     NodeID GetLastEdgeTargetID(const EdgeID edge_id) const;
     NodeID GetLastEdgeSourceID(const EdgeID edge_id) const;
 
+    // Invalidates the internal storage
+    SegmentDataContainer ToSegmentData();
```

Comment:
Urgh this is a bit weird - can you do the following in the function impl.

    void fn() {
      static bool dead = false;
      assert !dead;
      dead = true;
      return move(data);
    }